packages/frontpage/app/(app)/moderation/_components/user-handle.tsx

```diff
@@ -0,0 +1,13 @@
+import { DID } from "@/lib/data/atproto/did";
+import { getVerifiedHandle } from "@/lib/data/atproto/identity";
+import Link from "next/link";
+
+export async function UserHandle({ userDid }: { userDid: DID }) {
+  const handle = (await getVerifiedHandle(userDid)) ?? userDid;
```

Comment:
It's possible for the handle to not be verified and therefore `getVerifiedHandle` to return null. Better to show the did rather than nothing.

Ideally I think we'd show the handle even if it's not verified, but we don't have any utils for doing that right now lol.